docs/openstack-openedx/supported-deployment.md

```diff
@@ -0,0 +1,99 @@
+  - Public cloud providers are listed in the [OpenStack Marketplace](https://www.openstack.org/marketplace/public-clouds/).
+  - At a minimum, you will need a VM with 4 GB of RAM
+* A MySQL server/cluster running MySQL version 5.6 or newer
+* A MongoDB server/cluster running MongoDB version 2.6, 3.0, or 3.2
+* A VM running RabbitMQ
+* A VM running ElasticSearch version 0.90.13 (not any newer version) - may be installed on the * same VM as RabbitMQ
+* An SSL certificate for each of the domains (typically there are three: www.example.com, studio.example.com, preview.example.com). We recommend using Let's Encrypt for free SSL certificates.
+ 
+As these are standard open source services, **the installation, configuration, and operation of these database servers is left to your operations team**, and would not be supported by OpenCraft nor edX. However, ansible playbooks which may be used to set up most of these services are provided at https://github.com/edx/configuration/tree/master/playbooks (edX) and https://github.com/open-craft/deployment-deploy-databases (OpenCraft).
+ 
+Additional notes:
+* All of the databases and services like RabbitMQ and ElasticSearch should be firewalled off from the public internet.
+* Be sure to implement some sort of regular backup for the data in MySQL, MongoDB, and Swift.
+
+
+## How to deploy for the first time
+1. Provision an OpenStack VM that runs Ubuntu 16.04 and has at least 4 GB of RAM. Ensure that ports 22, 80, and 443 are open to the internet.
+1. Provision a Swift container
+1. Set up MySQL:
+   1. Provision five MySQL databases:
+       - edxapp
+       - xqueue
+       - edxapp_csmh
+       - edx_notes_api
+       - notifier
+   1. For each MySQL database, create a username and password with read-write access to the database (for best security, do not allow these users to have schema change permission)
+   1. Create two global users with access to all databases: one, a "migration" user with schema change permission on all databases, and one "read only" user with no change/write permissions.
+1. Set up MongoDB: Create two databases - one for courseware and one for forum posts. Create a separate username and password for each database.
+1. Set up RabbitMQ: Create a username+password for "celery" and another username+password for "xqueue".
+1. Set up ElasticSearch as listed in the requirements section above.
+1. Create a private git repository (we recommend using GitHub or GitLab) with a name like "sitename-vars" where sitename is the name of your new Open edX site
+1. In that new repository, create a vars.yml file using [this template](https://github.com/open-craft/opencraft/pull/204) (or [this alternate template](https://github.com/open-craft/openedx-deployment/blob/supported-deployment-template/openstack-openedx/vars-template.yml)) as a template
```

Comment:
Can we only mention the first one, since we wouldn't want to maintain the second one?